src/database/queries/ApprovedItem.ts

```diff
@@ -206,9 +206,9 @@ export async function getScheduledSurfaceHistory(
     return {
       ...item,
       // Format the scheduled date to YYYY-MM-DD format
-      scheduledDate: DateTime.fromJSDate(item.scheduledDate).toFormat(
-        'yyyy-MM-dd'
-      ),
+      scheduledDate: DateTime.fromJSDate(item.scheduledDate, {
+        zone: 'utc',
```

Comment:
trying to understand why we are doing this - if we set the timezone in the sql docker, then it by default considers all date in utc format right?

Reply:
This is bug prevention for cases where the system timezone is not UTC. This keeps tests working well on developer machines outside containers, and is just extra insurance that we do not have off by one date issues if somehow this accidentally gets deployed in a system with a non-utc timezone.

Reply:
There was more conversation around this in slack, putting a quick reproduction snippet here showing the exact thing we're trying to prevent here.  In a node REPL on a system with a negative UTC offset:
```
> const scheduledDate = new Date('2050-01-01T00:00:00.000Z')
undefined
> scheduledDate.toLocaleString('en-US', { dateStyle: 'medium' })
'Dec 31, 2049'
> scheduledDate.toLocaleString('en-US', { dateStyle: 'medium', timeZone: 'UTC' })
'Jan 1, 2050'
```

The luxon DateTime behaves identically.

Reply:
Both dates say `'Jan 1, 2050'` for systems that are UTC or have a positive UTC offset as well.